sql/core/src/test/scala/org/apache/spark/sql/execution/BaseScriptTransformationSuite.scala

```diff
@@ -372,6 +372,35 @@ abstract class BaseScriptTransformationSuite extends SparkPlanTest with SQLTestU
             'e.cast("string"))).collect())
     }
   }
+
+  test("SPARK-32667: SCRIPT TRANSFORM pad null value to fill column" +
+    " when without schema less (no-serde)") {
```

Comment:
please update "no-serde" in the codebase in your other TRANSFORM PRs.

Reply:
> please update "no-serde" in the codebase in your other TRANSFORM PRs.

Will raise a pr to handle this together